Detectors/MUON/MCH/Simulation/include/MCHSimulation/Response.h

```diff
@@ -30,50 +30,71 @@ class Response
   Response(Station station);
   ~Response() = default;
 
-  float getQspreadX() { return mQspreadX; };
-  float getQspreadY() { return mQspreadY; };
-  float getChargeSat() { return mChargeSat; };
-  float getChargeThreshold() { return mChargeThreshold; };
+  const float getQspreadX() { return mQspreadX; };
```

Comment:
@mwinn2 the idea was to make the _method_ const, not the return value

Reply:
@aphecetche ok, done.